tests/TestCase.php

```diff
@@ -58,9 +58,9 @@ protected function getEnvironmentSetUp($app)
                 'driver' => 'pgsql',
                 'host' => 'testing',
                 'port' => '5432',
-                'database' => 'homestead',
-                'username' => 'homestead',
-                'password' => 'secret',
+                'database' => env('DB_DATABASE', 'laravel'),
```

Comment:
Why?

Reply:
This code we introduced in admin-generator in this commit https://github.com/BRACKETS-by-TRIAD/admin-generator/commit/e39777a16be3054f89f2dfe56b26999a16f4d464#diff-6f6bda9a500f247548c1b366aa0d344a because in Laravel 6 default database name is ```laravel``` instead of ```homestead``` like in older versions of Laravel